src/fileformats/ocd_file_import.h

```diff
@@ -46,8 +46,15 @@
 #include "fileformats/ocd_types_v8.h" // IWYU pragma: keep
 
 class QChar;
+class QImage;
 class QTextCodec;
 
+namespace Ocd {
+
+struct IconV9;
```

Comment:
How come that only `IconV9` has to be forward-declarated?

Reply:
Because the `ocd_types_v8.h` header is already included, line 46.